android/src/main/java/com/swmansion/reanimated/nodes/OperatorNode.java

```diff
@@ -137,15 +137,15 @@ public double evaluate(Node[] input) {
   };
   private static final Operator NOT = new Operator() {
     @Override
-    public double evaluate(Node[] input) {
+    public Object evaluate(Node[] input) {
       return truthy(input[0].value()) ? 0. : 1.;
     }
   };
   private static final Operator DEFINED = new Operator() {
     @Override
-    public double evaluate(Node[] input) {
+    public Object evaluate(Node[] input) {
```

Comment:
can we special case DEFINED in `evaluete` and leave out all the changes

Reply:
indeed i figured out there's no need for any special case as the result of `defined` is double